Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -164,6 +182,25 @@ + (void)clearConfiguration
     globalConfiguration() = nil;
 }
 
++ (void)maybeConsumeBundlePaths:(NSDictionary<NSAttributedStringDocumentReadingOptionKey, id> *)options
+{
+    auto* readAccessPaths = dynamic_objc_cast<NSArray<NSURL *>>(options[WKReadAccessFileURLsOption]);
+    if (!readAccessPaths)
+        return;
+
+    if (!readAccessPaths.count)
+        return;
```

Comment:
Same question. But why add a special case at all? Does it do harm to accept an empty array?

Reply:
It's not harmful -- it just return to the default state of no additional directories being sent to the WebContent process. I'll revise to handle that, and add a test case.